experimental/dds/tree2/src/simple-tree/rawNode.ts

```diff
@@ -0,0 +1,276 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FieldKey, TreeNodeSchemaIdentifier } from "../core";
+import {
+	EditableTreeEvents,
+	FieldNodeSchema,
+	FlexTreeContext,
+	FlexTreeField,
+	FlexTreeNode,
+	FlexTreeTypedNode,
+	FlexTreeEntityKind,
+	FlexTreeMapNode,
+	FlexTreeObjectNode,
+	FlexTreeTypedField,
+	FlexTreeUnboxField,
+	FlexibleFieldContent,
+	boxedIterator,
+	flexTreeMarker,
+	onNextChange,
+	LocalNodeKey,
+	MapNodeSchema,
+	ObjectNodeSchema,
+	TreeNodeSchema,
+	TreeNodeSchemaBase,
+	TreeStatus,
+	FlexTreeFieldNode,
+} from "../feature-libraries";
+import { fail } from "../util";
+import { InsertableTypedNode } from "./insertable";
+
+/** Stores the content of the raw node, i.e. the data that was passed to the factory */
+const nodeContent = Symbol();
+interface HasNodeContent<T> {
+	[nodeContent]: T;
+}
+
+/**
+ * Retrieve the content of a raw object node created via {@link createRawNode}.
```

Comment:
Nit: probably worth noting that this actually removes the content from the input object

Reply:
I clarified the remarks.